### PR TITLE
documentation update - Release 3.7.0

### PR DESCRIPTION
**Pull request by Saksham-Nautiyal**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, pep8, linting, or security issues
- [ ] I have performed Ansible Sanity test using --docker default
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: docs/RELEASE_NOTES.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Table 1. Revision history
 
 | Revision | Date         | Description                                               |
 |----------|--------------|-----------------------------------------------------------|
-| 01       | November 2024 | Current release of Ansible Modules for Dell PowerStore 3.6.0 |
+| 01       | December 2024 | Current release of Ansible Modules for Dell PowerStore 3.6.0 |
```
Comment (sachin-apa) [inline docs/RELEASE_NOTES.md:31]:
```suggestion
| 01       | December 2024 | Current release of Ansible Modules for Dell PowerStore 3.7.0 |
```

**Review by sachin-apa — DISMISSED** [review]

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,6 +4,15 @@ Dellemc.Powerstore Change Logs
...
 .. contents:: Topics
 
+v3.7.0
+======
+
+Minor Changes
+-------------
+
+- Host module is enhanced to support check and diff mode.
+- Volume module is enhanced to support check and diff mode.
```
Comment (trisha-dell) [inline CHANGELOG.rst:14]: Add bug fix for volume module as well.
"Added support to modify size and host in the same task"

File: changelogs/changelog.yaml [hunk 3]
```diff
@@ -274,3 +274,9 @@ releases:
         name: snmp_manager
         namespace: ''
     release_date: '2024-11-29'
+  3.7.0:
+    changes:
+      minor_changes:
+        - Host module is enhanced to support check and diff mode.
+        - Volume module is enhanced to support check and diff mode.
```
Comment (trisha-dell) [inline changelogs/changelog.yaml:281]: update bug fix here as well, applicable to all such instances.

File: docs/RELEASE_NOTES.md (hunk 1 above)
Comment (trisha-dell) [inline docs/RELEASE_NOTES.md:31]: PoweStore 3.7.0

File: changelogs/changelog.yaml (hunk 3 above)
Comment (Saksham-Nautiyal) [reply changelogs/changelog.yaml:281]: done

File: CHANGELOG.rst (hunk 2 above)
Comment (Saksham-Nautiyal) [reply CHANGELOG.rst]: done

File: docs/RELEASE_NOTES.md (hunk 1 above)
Comment (Saksham-Nautiyal) [reply docs/RELEASE_NOTES.md]: done

**Review by ShivamSh3 — APPROVED** [review]

**Review by sachin-apa — APPROVED** [review]

**Review by sachin-apa — APPROVED** [review]

**Review by ABHISHEK-SINHA10 — APPROVED** [review]

**Review by trisha-dell — APPROVED** [review]

**Pull request merged**